CompositeUi/src/views/component/SearchMember.js

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
SearchMember component is identical to the Search component so, you can remove safely